kafka/README.md

```diff
@@ -13,6 +13,10 @@ This check has a limit of 350 metrics per instance. The number of returned metri
 
 To collect Kafka consumer metrics, see the [kafka_consumer check][3].
 
+*Note*: This integration attached sample configuration works only for Kafka >= 0.8.2.
```

Comment:
I didn't realize that the legacy integration was for such old versions of kakfa. 0.10 was EOL in Oct 2016